frontend/src/components/AppPercentage.vue

```diff
@@ -3,7 +3,7 @@
 -->
 
 <template>
-  <div v-tooltip="tooltip">
+  <div v-tooltip="tooltip" role="tooltip" aria-hidden="true">
```

Comment:
A few things about this:

- Role is meant to give accessible meaning to an element. Aria-hidden hides an element from the accessibility tree. As such, this really serves to quiet the error and bypass the test. Aria-hidden is appropriate when something is presentational, but this tooltip is not.
- The tooltip role is meant for the box that is actually the content of the tooltip, not the trigger. The reason the e2e axe tests are complaining is that div has no semantic meaning and is also not inherently interactive (e.g. can't tab to it). The way we've dealt with this in other places in the app (like the text annotator, where the results are just spans that have tooltips) is to assign `tabindex="0"` so you can tab to them and a person using a screen reader could access the tooltip.
- I strongly recommend always minimizing the number of elements you put in the DOM, to minimize "div hell" and css positioning and selector complexity. Here you have a div simply to avoid putting the same `v-tooltip` on two elements, which there's a different way to avoid.

Reply:
Gotcha yeah, was definitely not sure about that "fix", I appreciate the detailed feedback! 

So step one would be replacing the `aria-hidden` with `tabindex="0"`. 

And I can get rid of the unnecessary outer div, but what would be the way to avoid adding `v-tooltip="tooltip" tabindex="0"` on both the inner divs?

Guess I could move the `v-if` to the different svg elements and replace `class="ring/bar"` with `:class="type"`, unless you had something else in mind

Reply:
In the past for things like this I've used a role of `button`, but perhaps `note` or `definition` would actually be better, I don't really know. Here's an SO post on the subject:

https://stackoverflow.com/questions/70742579/what-is-the-correct-aria-role-for-a-more-info-icon

We don't want role `img` or `presentational` or anything like that because I believe then screen readers would ignore reading the number text as well.

In fact, just using `<meter>` semantically here might even be better, but then we also have to give it min and max values which could be a pain (or useful).

Reply:
> what would be the way to avoid adding v-tooltip="tooltip" tabindex="0" on both the inner divs?

I'd honestly recommend duplicating some props in your template over duplicating actual divs in the browser. However in this particular case there was an easy way around it. If you had a situation when you really needed two root elements and needed to pass a lot of the same props to both of them, you could just group them together down in the `<script>` and pass them in via `v-bind`.

Reply:
Gotcha, that makes sense, so something like [this SO post](https://stackoverflow.com/a/46731340). 

I was also just about to push a change I made locally before I noticed your commit. 
Would this also be a viable approach, or is it code-smelly for its own reasons? 
```ts
<template>
  <div :class="type" v-tooltip="tooltip" tabindex="0" role="tooltip">
    <svg v-if="type === 'ring'" viewBox="-50 -50 100 100">
      <circle class="back" cx="0" cy="0" r="50" />
      <path class="fill" :d="arc" />
    </svg>
    <b>
      <slot />
    </b>
    <svg v-if="type === 'bar'" viewBox="0 0 70 10">
      <rect class="back" x="0" y="0" width="100%" height="100%" />
      <rect
        class="fill"
        x="0"
        y="0"
        :width="`${100 * percent}%`"
        height="100%"
      />
    </svg>
  </div>
</template>
```

Reply:
Oh right right, ok. Sounds good, thanks!